### ISSUE TITLE
When will it be safe to turn off read-only mode

### ISSUE THREAD

**Issue by BillH99999** [zanonmark/Google-4-TbSync#50]
It still says the following on the web site.

"You are strongly suggested to use the [Read-only mode option](https://github.com/zanonmark/Google-4-TbSync/wiki/Account-options#read-only-mode)."

Do you have any idea of when it will be safe to turn off read only mode?

Thanks,
Bill

**Comment by zanonmark** [comment]
Well, we have two issues here:

1) The maturity of the project.
Google-4-TbSync is still quite young, and it's still undergoing major changes.
I think a completely new version will be released in mid-spring, where we'll use the new WebExtensions API in Thunderbird, simplifying and speeding up things and adding some missing features too, but also requiring the engine to be half-rewritten (again!).

2) The data structure.
Thunderbird has an addressbook structure which is quite different from the Google's one. Recent versions (102+) have made some steps forward, but still some fields are simply different from Google. Importing from and then exporting to Google could result in data lost or scrambled.

So, basically: until at least summer 2023, the code cannot be considered "stable", and then... we shall see :)

Thanks,
MZ

**Comment by BillH99999** [comment]
OK... thanks Marco.  I am looking forward to the new version.

Thanks for all the hard work.  

Bill

**Comment by zanonmark** [comment]
Thanks,
MZ
